modis_tools/constants/urls.py

```diff
@@ -8,5 +8,6 @@ class URLs(Enum):
     API: str = "cmr.earthdata.nasa.gov"
     URS: str = "urs.earthdata.nasa.gov"
     RESOURCE: str = "e4ftl01.cr.usgs.gov"
+    MOD11A2_V061_RESOURCE: str = "data.lpdaac.earthdatacloud.nasa.gov"
```

Comment:
Eddy, did you have a chance to test products other than MOD11A2? Just checking whether we can add this as a more general resource for v061 products